src/index.js

```diff
@@ -12,7 +12,7 @@ class RetryChunkLoadPlugin {
       const { mainTemplate } = compilation;
       if (mainTemplate.hooks.jsonpScript) {
         // Adapted from https://github.com/webpack/webpack/blob/11e94dd2d0a8d8baae75e715ff8a69f27a9e3014/lib/web/JsonpMainTemplatePlugin.js#L145-L210
-        mainTemplate.hooks.jsonpScript.tap(pluginName, () => {
+        mainTemplate.hooks.jsonpScript.tap(pluginName, (_, chunk) => {
```

Comment:
So if you change `_` to `scriptWithoutRetry` and remove `scriptWithoutRetry` below then it should do the same thing with less code 😄

Reply:
Awesome :) Have made the change.